incubator/elasticsearch/README.md

```diff
@@ -31,66 +31,122 @@ The command deploys Elasticsearch on the Kubernetes cluster in the default confi
+| `ingest.livenessProbe.timeoutSeconds`             | When the probe times out (ingest nodes pod)                                                                               | `5`                                                                 |
+| `ingest.livenessProbe.successThreshold`           | Minimum consecutive successes for the probe to be considered successful after having failed (ingest nodes pod)            | `1`                                                                 |
+| `ingest.livenessProbe.failureThreshold`           | Minimum consecutive failures for the probe to be considered failed after having succeeded                                 | `5`                                                                 |
+| `ingest.readinessProbe.enabled`                   | Enable/disable the readiness probe (ingest nodes pod)                                                                     | `true`                                                              |
+| `ingest.readinessProbe.initialDelaySeconds`       | Delay before readiness probe is initiated (ingest nodes pod)                                                              | `90`                                                                |
+| `ingest.readinessProbe.periodSeconds`             | How often to perform the probe (ingest nodes pod)                                                                         | `10`                                                                |
+| `ingest.readinessProbe.timeoutSeconds`            | When the probe times out (ingest nodes pod)                                                                               | `5`                                                                 |
+| `ingest.readinessProbe.successThreshold`          | Minimum consecutive successes for the probe to be considered successful after having failed (ingest nodes pod)            | `1`                                                                 |
+| `ingest.readinessProbe.failureThreshold`          | Minimum consecutive failures for the probe to be considered failed after having succeeded                                 | `5`                                                                 |
+| `metrics.enabled`                                 | Enable prometheus exporter                                                                                                | `false`                                                             |
+| `metrics.name`                                    | Metrics pod name                                                                                                          | `metrics`                                                           |
+| `metrics.image.name`                              | Metrics exporter image name                                                                                               | `justwatch/elasticsearch_exporter`                                  |
```

Comment:
To make the interface homogeneous I think we should adopt the following. 

```
metrics:
  image:
    registry: docker.io
    repository: justwatch/elasticsearch_exporter
    tag: 1.0.1
    pullPolicy: IfNotPresent
```

Reply:
I agree 👍